### PR TITLE
feat: Created homepage of fakeSea 🚀

### PR DESCRIPTION
**Pull request by arjunbharti**
Created Homepage of fakeSea

- Header
- Main layout
- Trending section
- Buyer's guide
- Footer
- Responsiveness

Preview: https://deploy-preview-3--fakesea.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,10 +9,295 @@
 
     <!-- external imports -->
     <link rel="icon" href="./assets/icon.png" type="image/icon">
+    <link rel="stylesheet" href="./styles/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
```
Comment (viru2001) [inline index.html:14]: import this in your component library css file

File: styles/homepage.css [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+    max-height: 3rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 11rem;
+}
+
+.logo{
+    width: 40px;
+    height: 40px;
```
Comment (viru2001) [inline styles/homepage.css:21]: you can use relative units here

File: styles/utils.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
+:root{
+    --primary-color: #60A5FA;
+    --primary-hover: #3B82F6;
+    --secondary-color: #BFDBFE;
+    --secondary-hover: #93C5FD;
+    --background: rgb(237, 242, 252);
+    --danger: lightcoral;
+    --warning: yellow;
+    --success: lightgreen;
+    --light: white;
```
Comment (viru2001) [inline styles/utils.css:10]: use only hex or only rgb()

**Review by viru2001 — COMMENTED** [review]
UI looks good . 
some suggestions from my side :
1. you should give `cursor: pointer` to buttons in the card
2. you can add box-shadow to cards only on hovering over them

**Review by arjunbharti — COMMENTED** [review]
Was going through my code

File: index.html (hunk 1 above)
Comment (arjunbharti) [reply index.html:14]: Already mentioned in component library. I will delete this here. Thanks for pointing this out

File: styles/homepage.css (hunk 2 above)
Comment (arjunbharti) [reply styles/homepage.css:21]: Changed successfully.

**Pull request merged**